### ISSUE TITLE
Process randomly stops on Parler

### ISSUE THREAD

**Issue by martinratinaud** [ambanum/OpenTermsArchive#669]
Here are the logs that were issued after this problem

```
2021-11-06 10:35:22 error Parler — Privacy Policy                                 FetchError: request to https://parler.com/documents/privacypolicy.pdf failed, reason: unable to verify the first certificate
    at ClientRequest.<anonymous> (file:///home/debian/cgus/node_modules/node-fetch/src/index.js:94:11)
    at ClientRequest.emit (events.js:400:28)
    at TLSSocket.socketErrorListener (_http_client.js:475:9)
    at TLSSocket.emit (events.js:400:28)
    at emitErrorNT (internal/streams/destroy.js:106:8)
    at emitErrorCloseNT (internal/streams/destroy.js:74:3)
    at processTicksAndRejections (internal/process/task_queues.js:82:21)
2021-11-06 10:35:22 error                                                         unhandledRejection: request to https://parler.com/documents/privacypolicy.pdf failed, reason: unable to verify the first certificate
FetchError: request to https://parler.com/documents/privacypolicy.pdf failed, reason: unable to verify the first certificate
    at ClientRequest.<anonymous> (file:///home/debian/cgus/node_modules/node-fetch/src/index.js:94:11)
    at ClientRequest.emit (events.js:400:28)
    at TLSSocket.socketErrorListener (_http_client.js:475:9)
    at TLSSocket.emit (events.js:400:28)
    at emitErrorNT (internal/streams/destroy.js:106:8)
    at emitErrorCloseNT (internal/streams/destroy.js:74:3)
    at processTicksAndRejections (internal/process/task_queues.js:82:21)
2021-11-06 10:35:22 error                                                         unhandledRejection FetchError: request to https://parler.com/documents/privacypolicy.pdf failed, reason: unable to verify the first certificate
2021-11-06 10:35:22 error                                                         request to https://parler.com/documents/privacypolicy.pdf failed, reason: unable to verify the first certificate
error: Forever detected script exited with code: 1
```

**Comment by martinratinaud** [comment]
I restart the server now.

**Comment by martinratinaud** [comment]
It seems the error comes from the fact that we receive a `UNABLE_TO_VERIFY_LEAF_SIGNATURE` error

Although, when I directly access the pdf file, I can't seem to get this error https://parler.com/documents/privacypolicy.pdf

I believe we should add `UNABLE_TO_VERIFY_LEAF_SIGNATURE` to the list of handled errors as it is also a certificate error

WDYT @Ndpnt

**Comment by MattiSG** [comment]
Fixed by #657.
